src/common/components/survey-form/survey-form.component.less

```diff
@@ -0,0 +1,15 @@
+.edit-survey__form {
+  display: flex;
+  flex-direction: column;
+  max-width: 500px;
+  margin: 0 auto;
```

Comment:
Styling leaves something to be desired but will do for now

Reply:
Agreed, it's a bit shit but since this has to be moved to modal I quick fixed it